qcfractal/interface/models/rest_models.py

```diff
@@ -8,25 +8,80 @@
 from .common_models import KeywordSet, Molecule, ObjectId
 from .gridoptimization import GridOptimizationInput
 from .model_utils import json_encoders
+from .records import ResultRecord
 from .task_models import TaskRecord
 from .torsiondrive import TorsionDriveInput
-from .records import ResultRecord
 
 __all__ = [
```

Comment:
Since you've refactored this to reduce the number of things needed to be imported, it would be good to also change the `__all__` to be lest complicated now. From the `client` file, the `__all__` should now be 
`[ComputeResponse, rest_model, QueryStr, QueryObjectId, QueryProjection]`, unless you want to also expose all of the `GET`, `PUT`, and `POST` methods still?